### ISSUE TITLE
Create wireframes for updated /web-literacy with Web Lit 2.0 design

### ISSUE THREAD

**Issue by sabrinang** [mozilla/teach.mozilla.org#1603]
Design wireframes that layout of the final Web Lit 2.0 design in the context of teach.mozilla.org

- Figure out how curriculum is connected with [Web Lit Basics I](https://docs.google.com/a/mozilla.com/document/d/15B2ccF-DSTl74SwNbyZ0W5qMu-jQNOXUS4C6BsvjF9s/edit?usp=sharing) 
- Figure out how badges are connected
- ask for advice on localization system with this and curriculum
- consider request for making Web Literacy 2.0 a primary navigation item - IA (currently a secondary item under teaching activities)

Content dependent on issues: #1600 (whitepaper link), #1601 (design), #1602 (copy)
cc/ @anmechung

**Comment by anmechung** [comment]
@chadsansing isn't there a github ticket already where we determined that the Web LIteracy 2.0 would be a primary navigation item?

**Comment by hannahkane** [comment]
Would recommend pulling out localization as a separate ticket (and wouldn't include localization of curriculum, which is happening through another process).

I think Web Literacy as a primary nav item is OK (but wouldn't add "2.0")

**Comment by cassiemc** [comment]
Seems like we don't need to figure out badges yet (because our current badge stuff isn't live)?

**Comment by hannahkane** [comment]
Agreed re: @cassiemc's point above. I think we can limit this ticket to the current first and fourth bullet points. 

Re: the navigational hierarchy. Let's make this a top-level item, below Activities and above "Leadership Opportunities." That means the page will need a hero image and copy. I suggest "Web Literacy" for the title and "Reading, writing, and participating on the open web" for the sub-title

**Comment by anmechung** [comment]
+1 @hannahkane

**Comment by jgmac1106** [comment]
Yeah finally not a subnavigation menu. Me soo happy.

**Comment by sabrinang** [comment]
**Updates:**
- Web Lit Basics I activities are connected to the corresponding literacy or competency selected in the graph by being displayed below the description as a thumbnail list (as seen on teaching activities page)
- Web Literacy is a primary navigation item and 21 Century Skills is a secondary item beneath this
- whitepaper will be linked from the bottom of the page 
- wireframes show various selected states and responsive views

Wireframes are located in this redpen: https://redpen.io/p/to351d853022563354

**Comment by sabrinang** [comment]
cc/ @cassiemc @hannahkane @anmechung 

Let's review these

**Comment by sabrinang** [comment]
Hey @anmechung - I believe you mentioned there is the department of ed training in New York gathering 3rd week of February and you wanted to share out web lit progress so here is a clickable wireframe (organized better than the redpen) on marvel for now: https://marvelapp.com/b8a125

(I think there are some typos I made copy and pasting but I am going through them now)

**Comment by jgmac1106** [comment]
@sabrinang this is awesome. I love the interactivity and how much info you
fit into such an intuitive design.

**Comment by anmechung** [comment]
@sabrinang this is just GREAT!!! Beautiful and functional. I added some editing comments to redpen, and what I will do when is go through each skill, cross-reference and make sure they match with what's in the paper

**Comment by hannahkane** [comment]
@sabrinang - do you think we can close this issue?

**Comment by sabrinang** [comment]
@hannahkane Yes - wireframes have reviewed yesterday and any small text adjustments will be done in dev phase. (I'll file separate issue for final line art and colour)
